server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -722,7 +707,7 @@ private UploadedMetadataResults writeMetadataInParallel(
         UploadedMetadataResults response = new UploadedMetadataResults();
         results.forEach((name, uploadedMetadata) -> {
             if (uploadedMetadata.getClass().equals(UploadedIndexMetadata.class)
-                && uploadedMetadata.getComponent().contains(InternalRemoteRoutingTableService.INDEX_ROUTING_METADATA_PREFIX)) {
+                && uploadedMetadata.getComponent().contains(INDEX_ROUTING_TABLE_PREFIX)) {
```

Comment:
We should use the same variable while adding the task and checking here. In case delimiter changes or prefix changes, this will break.

Reply:
Done

Reply:
While adding the task we are still using different variable